app/start-demo.sh

```diff
@@ -0,0 +1,4 @@
+ #!/bin/bash
+export MONGO_URL="mongodb://localhost:27019/wellbeing-demo"
+mongo wellbeing-demo --port 27019 --eval "db.dropDatabase()"
```

Comment:
I am getting an error related to this line:

```
$ mongo wellbeing-demo --port 27019 --eval "db.dropDatabase()"
MongoDB shell version v4.0.15
connecting to: mongodb://127.0.0.1:27019/wellbeing-demo?gssapiServiceName=mongodb
2020-01-25T10:42:29.040+0200 E QUERY    [js] Error: couldn't connect to server 127.0.0.1:27019, connection attempt failed: SocketException: Error connecting to 127.0.0.1:27019 :: caused by :: Connection refused :
connect@src/mongo/shell/mongo.js:344:17
@(connect):2:6
exception: connect failed
```

Reply:
Would it be possible to use the Meteor internal Mongodb rather than a system-level installation of Mongo? E.g. Meteor has the `meteor mongo` shell might be able to run a single command to drop the DB.

I have been able to get the local demo running by changing the shell script to the following:

```
export MONGO_URL="mongodb://localhost:27017/wellbeing-demo"
meteor
```

Reply:
Meteor reset should be fine here to reset the database:
https://docs.meteor.com/commandline.html#meteorreset

Reply:
Ah, except 

> The database specified with MONGO_URL will NOT be reset.

Reply:
I'm not sure how to approach this. Ideally the solution wouldn't require me to install any Mongodb dependencies on my local computer. It would also be possible to deploy a demo instance of GeriLife running on a cloud server. So, could we consider an approach that looks for a general `GERILIFE_DEMO=True` rather than relying on specifying Mongodb port and collection name?

Reply:
yes I also change the port. but only a different database name  should do. let me check if that locally first. However a separate database name will be required than the default one. and we need to set mongo URL for that

Reply:
OK, thanks. I am just having trouble working with Mongo locally, so I would like to rely on Meteor abstracting things to the extent possible.

It would be good also if this script could be deployed to a server and refreshed every night or hour. That way, people could try an online demo at their convenience.

Reply:
then in that case, it will be better to just given the demo environment variable and deploy a separateinstance or docker container. So we do not need to change db name.

For the second part, a cron can be written on the server that restarts every night.

Reply:
@brylie  Do you want to go for a separate instance for this?

Reply:
Yeah, the cron approach sounds fine. Just look for the `GERILIFE_DEMO` environment variable and reset the database by cron nightly.

Reply:
Yes I have already changed it to gerilife_demo. For writing the cron I need know the platform of deployment. Will it be linux based?

We will need a separate container or something cannot deploy it along with the production or else it will write over the production database as by default it start mongodb at 3001

Reply:
Could we use Dokku for deployment?

http://dokku.viewdocs.io/dokku/

Dokku relies on standard Docker images or optionally a buildpack
- http://dokku.viewdocs.io/dokku/deployment/methods/dockerfiles/
- http://dokku.viewdocs.io/dokku/deployment/methods/buildpacks/

Reply:
looks good. just working out some details to run meteor properly in cron environment

Reply:
> This works great. Just go ahead and add instructions to the README for running the demo app. Make sure to rebase any changes, if necessary.

done